clientv3/balancer.go

```diff
@@ -26,10 +26,17 @@ import (
 // simpleBalancer does the bare minimum to expose multiple eps
 // to the grpc reconnection code path
 type simpleBalancer struct {
-	// eps are the client's endpoints stripped of any URL scheme
-	eps     []string
-	ch      chan []grpc.Address
+	// addrs are the client's endpoints for grpc
+	addrs []grpc.Address
+
+	ch chan []grpc.Address
+
+	// upEps contains the endpoints grpc says are up
+	upEps map[string]struct{}
+	// numGets chooses the next endpoint
 	numGets uint32
+	// mu protects upEps, downEps, and numGets
```

Comment:
Maybe `mu sync.Mutex` on the top of upEps and numGets?

And what is `downEps`? I don't see it in the struct fields?